wagtail/users/wagtail_hooks.py

```diff
@@ -50,7 +43,10 @@ def get_group_viewset_cls(app_config):
 def register_viewset():
     app_config = apps.get_app_config("wagtailusers")
     group_viewset_cls = get_group_viewset_cls(app_config)
-    return group_viewset_cls("wagtailusers_groups", url_prefix="groups")
+    return [
+        UserViewSet("wagtailusers_users", url_prefix="users"),
```

Comment:
Would be nice if developers could also customise the viewset class to use here, similar to the custom `GroupViewSet` support added in #6477. I think it makes even more sense for `UserViewSet`, as developers can already use a custom `User` model, so it must be pretty common to have custom logic that would need to be considered in the admin views.